web-client/src/modules/timeline/components/OffersList/OffersList.tsx

```diff
@@ -18,6 +20,7 @@ interface OffersListProps {
 interface OfferItemProps {
   offer: Offer;
```

Comment:
the data type of offer here should be changed to `OfferWithLocation` (the class abstract class from the backend which combines the details of the offer and the location/address).
This same change should happen in the `OffersListProps` in line 15 as well. there it should be `Record<string, OfferWithLocaiton>`

Reply:
oh, my bad.